samples/boards/stm32/usbc/sink/sample.yaml

```diff
@@ -1,7 +1,7 @@
 sample:
-  name: USBC SINK
+  name: BOARD STM32 USBC SINK
 tests:
-  sample.usbc.sink:
+  sample.board.stm32.usbc.sink:
```

Comment:
Now that you introduced a generic USBC Sink sample, should we keep this sample ?
If yes, would you mind document in what these samples differ ?

Reply:
I think the board specific sample should be removed.